package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "eth-packages",
+  "license": "MIT",
+  "scripts": {
+    "format:check": "prettier --check .",
+    "lint": "eslint --cache .",
+    "test": "npx lerna run test --concurrency 1 --stream",
+    "test:changed": "npm run test -- --since master"
+  },
+  "dependencies": {
+    "lerna": "^7.0.2"
+  },
+  "devDependencies": {
+    "@types/chai-as-promised": "7.1.8",
+    "@types/mocha": "10.0.6",
+    "@types/sinon": "17.0.3",
+    "@typescript-eslint/eslint-plugin": "7.8.0",
+    "@typescript-eslint/parser": "7.8.0",
+    "chai": "4.4.1",
+    "chai-as-promised": "7.1.1",
+    "eslint": "^8.43.0",
+    "eslint-config-bloq": "^4.0.2",
+    "husky": "^4.3.8",
+    "lint-staged": "^13.2.2",
+    "mocha": "10.4.0",
+    "prettier": "^3.2.5",
+    "sinon": "17.0.1"
+  },
+  "workspaces": [
+    "packages/*"
+  ],
+  "eslintConfig": {
+    "extends": [
+      "bloq",
+      "plugin:@typescript-eslint/eslint-recommended",
+      "plugin:@typescript-eslint/recommended",
+      "prettier"
+    ],
+    "ignorePatterns": "packages/*/dist/*",
+    "overrides": [
+      {
+        "files": [
+          "*.spec.js"
+        ],
+        "extends": [
+          "prettier"
+        ],
```

Comment:
Why are the rules overwritten here? We should have added `bloq`, `mocha` and then `prettier`, I think.

Reply:
the problem was that `bloq` and `mocha` work only with commonjs and not with `esm` (and I was unable to mix js commonjs in the test with `esm` from typescript)

We could check it together, but the simplest option I found was to move the test to use `esm` (though still in js). Otherwise, we'd need to include another tool just for the tests 🤔

Reply:
I think I was able to make it work here [00e4352](https://github.com/bloq/eth-packages/pull/1/commits/00e435291a3e4189dfb3872eec43c1e08f8ff95e) but only by disabling the rule `node/no-unpublished-require` (which is only for commonjs)